### PR TITLE
Add documentation about GitHub Actions usage

### PR DESCRIPTION
**Pull request by SeanTAllen**

### CODE REVIEW

File: ci/gh-actions-security.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+Most of the executed code is created and published by either us or GitHub. We also use some actions that are published by additional 3rd parties.
+
+## Our Rules for GitHub Actions
+
+- PRs are not allowed to run actions without approval unless the person opening the PR is a member of the Ponylang organization
+- Only actions from GitHub's `actions` organization and our own organization can run using a Docker image or references a git branch or tag
+- All "additional 3rd party" actions **MUST** only for a specific git commit hash
+- Only "additional 3rd party" actions that have been reviewed and considered trustworthy can be used
+- Only Docker images that have been blessed by DockerHub or GitHub can be used as base images for images that we use as part of our CI process if the image is referenced by a tag
```
Comment (jemc) [inline ci/gh-actions-security.md]: Would be nice to be more explicit about what "blessed" means here.

Comment (SeanTAllen) [reply ci/gh-actions-security.md]: "vouched for by"?

I'm not sure what the proper term should be.

**Review by chalcolith — APPROVED** [review]

File: ci/gh-actions-security.md (hunk 1 above)
Comment (jemc) [reply ci/gh-actions-security.md]: I don't actually understand what specifically you mean by "blessed". It's not so much that I think you chose the wrong words, but I just don't understand the mechanics of what specific thing I would check to see if it was "blessed" or not.

Are you referring to like the "Official Images" section of DockerHub? Just trying to see what the specific test would be to know if something qualified here or not.

Comment (SeanTAllen) [reply ci/gh-actions-security.md]: Yes. Official images.

Comment (jemc) [reply ci/gh-actions-security.md]:
```suggestion
- Only Docker images that have been marked by DockerHub or GitHub as Official Images can be used as base images for images that we use as part of our CI process if the image is referenced by a tag
```

**Review by jemc — APPROVED** [review]

**Pull request merged**